ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/NewMigrationCommand.java

```diff
@@ -31,7 +35,11 @@
 
   @Override
   public void run() {
-    throw new UnsupportedOperationException();
+    try {
+      Files.createDirectories(Paths.get(projectPath));
+      new File(projectPath + "/ksql-migrations.properties").createNewFile();
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
+1 not only that, we should return correct error codes (using `System.exit`) if necessary so that it can be properly piped into scripts. I think getting error messages correct is a huge part of making this tool successful.

cc @colinhicks @spena  for their opinion on how this should work.

Reply:
I'd also recommend avoiding error stacks in the output that users see (logs can have the error messages). This turns people off from these tools and makes them look very rough around the edges.

Reply:
Yes, very important to return the correct error codes from the process. This is relevant for CI, too, where a non-zero code is expected to signal job step failure. We want to avoid forcing users to otherwise detect failure by parsing the output.

Reply:
+1. I  also usually avoid putting everything into one try/catch when I want to know what failed. This way I can display something like:
```
$ ksql-migrations new /unknown-dir/migrations-project
Creating directory: <project-dir>
Failed creating directory: "Directory /unknown-dir does not exist"
```
Which then you can return know what error code to return, like `System.exit(2)`. This number will let user know what's wrong when writing scripts.

When working with user tools, it is always recommended to display a good error message and an error status. They will thank you a lot if they write their on bash scripts to automate this tool.

Reply:
Done. For now, they all have the same error code, but after more parts of the tool are complete we can go through the errors and categorize them.